ARUHA-575 How to control partition count in Nakadi

lmontrieux opened the pull request:
This PR includes (slightly adapted) changes made to the nakadi readme. It adds the `default_statistics` object to the event type creation example, together with a short explanation.

Commit:
ARUHA-575 Add default statistics to sample event type creation

antban commented on docs/using/event-types.md:
```diff
@@ -68,6 +74,9 @@ The event type has a simple JSON Schema submitted as an escaped JSON string desc
 says events will be randomly allocated to partitions, and the owner's name is
 `"acme-order-service"`. The `enrichment_strategies` array says to apply `metadata_enrichment` to submitted events (common metadata is a feature of some categories).
 
+The event type has an optional `default_statistics` object, which controls the number of partitions. Nakadi will use a sensible default if no value is provided. The values provided here 
```
because of my mistake the field is called `default_sttistic`, not `default_statistics`

lmontrieux replied:
You mean `default_statistic`? 😆

lmontrieux replied:
Fixed it: https://github.com/zalando/nakadi-manual/pull/46/commits/561a4c1663f8f738c5ac95014c3911bd564cf858

v-stepanov replied:
@antban you did a typo in that word again :rofl:

Commit:
ARUHA-575 Fix `default_statistic` field name in description

v-stepanov commented on docs/using/event-types.md:
```diff
@@ -57,6 +57,12 @@ curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/event-t
+    "messages_per_minute": 1000,	
+    "message_size":	5,
+    "read_parallelism":	10,
+    "write_parallelism": 5
```
I'm afraid that users will use this by default and we will have 8 partitions for all event-types as a result.
What do you think?

lmontrieux replied:
@v-stepanov good point. We should probably stick to 1 partition by default. Should I change `read_parallelism` to `1`? I'll have to update the Nakadi readme as well.

v-stepanov replied:
yeah, I think it makes sense to have `read_parallelism` and `write_parallelism` as 1 here. Because some people will just copy/paste this example without too much thinking.

lmontrieux replied:
@v-stepanov https://github.com/zalando/nakadi-manual/pull/46/commits/7a146572490e5b0a3635a14a79cc77b4eae92544

Commit:
Merge branch 'master' into aruha-575

Commit:
ARUHA-575 Read and write parallelism set to 1

We don't want people to create 8 partitions if they don't need them, just because they copy/pasted
